src/search_options.hpp

```diff
@@ -431,51 +442,55 @@ void parseCommandLine(LambdaOptions & options, int argc, char const ** argv)
                           seqan3::value_list_validator{45, 62, 80});
     }
 
-    if (options.nucleotide_mode)
-        options.gapExtend = -2;
-    else
-        options.gapExtend = -1;
-
     parser.add_option(options.gapExtend,
                       '\0',
                       "score-gap",
                       "Score per gap character.",
                       seqan3::option_spec::advanced,
                       seqan3::arithmetic_range_validator{-1000, 1000});
 
-    if (options.nucleotide_mode)
-        options.gapOpen = -5;
-    else
-        options.gapOpen = -11;
-
     parser.add_option(options.gapOpen,
                       '\0',
                       "score-gap-open",
                       "Additional cost for opening gap.",
                       seqan3::option_spec::advanced,
                       seqan3::arithmetic_range_validator{-1000, 1000});
 
-    parser.add_section("Extension");
+    // parser.add_section("Extension");
 
-    //TODO this is only used in serial mode right now
-    parser.add_option(options.band,
-                      'b',
-                      "band",
-                      "Size of the DP-band used in extension (-3 means log2 of query length;"
-                      " -2 means sqrt of query length; -1 means full dp; n means band of size 2n+1)",
-                      seqan3::option_spec::advanced,
-                      seqan3::arithmetic_range_validator{-3, 1000});
-
-#if 0 //TODO make new guide
-    parser.add_section("Tuning");
-    parser.add_line("Tuning the seeding parameters and (de)activating alphabet "
-                    "reduction has a strong "
-                    "influence on both speed and sensitivity. We recommend the "
-                    "following alternative profiles for protein searches:", false);
-    parser.add_line("fast (high similarity):    --seed-delta-increases-length on", false);
-    parser.add_line("sensitive (lower similarity): --seed-offset 3", false);
+    // TODO this is only used in serial mode right now
+    // parser.add_option(options.band,
+    //                   'b',
+    //                   "band",
+    //                   "Size of the DP-band used in extension (-3 means log2 of query length;"
+    //                   " -2 means sqrt of query length; -1 means full dp; n means band of size 2n+1)",
+    //                   seqan3::option_spec::advanced,
+    //                   seqan3::arithmetic_range_validator{-3, 1000});
```

Comment:
If nothing uses this, I would propose to remove it (also the option itself). No point in carrying around more legacy codepaths.

Reply:
I guess depends if we ever want to introduce it again, that's why I left it. But we can remove it, just wanted to check.

Reply:
If we add it again, we can also add the option back. Considering other people (including future-me), I think it's better to reduce the amount of commented-out code!